e2e/aws_test.go

```diff
@@ -2,13 +2,70 @@ package e2e
 
 import (
 	"github.com/gravitational/robotest/e2e/framework"
-	"github.com/gravitational/robotest/e2e/specs"
+
+	"github.com/gravitational/robotest/e2e/uimodel"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
 )
 
 var _ = framework.RoboDescribe("AWS Integration Test", func() {
 	f := framework.New()
+	ctx := framework.TestContext
+
+	It("should provision a new cluster [provisioner:aws_install]", func() {
```

Comment:
yes, to give a structure. We tried this format  [provisioner:aws] [install][expand]
but decided to replace  [] with _ so you do not need to escape it with \\ when running focused tests. 
-  \\[provisioner:aws\\] \\[install\\]\\[expand\\]